docs/src/rules/no-unsafe-negation.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: no-unsafe-negation
 rule_type: problem
+handled_by_typescript:
+- 2358
+- 2360
+- 2365
```

Comment:
https://www.typescriptlang.org/play?#code/CYUwxgNghgTiAEEQBd4GsQE8Bc8DOyMAlgHYDmA3AFCiSwJKoD2ARgFa4DeAvlVXXjzwAwsiYx4PPkQBm8ABQBCDJnil4rNuGQBKSVXiH4AemMaADiBhQxE83DAhQJR-AC2UDEKKoQARwBXIgA3KCQSVDEFZSw9dU1tAyNTeCgSYHhkTEt4MCYSYKs8Inz3TxBvX0CQsJAIzKYFFXgAfngAIhkwvBB2+Fx2wgDeuJINdkTeKllozTUSAjTHJjlRcT1OJMMUpktrW3h7cCc61w8vNSqg0PDIxqVNUcWXEBWRWy2TMzSMn3h-a61epRMIAdygmCEXQgPXwpFcLCYTCQaXgN2GQno8BITGYEzAyDwADoqLwgA

* `2358` ✔️ 
* `2322`: _`Type 'boolean' is not assignable to type 'string | number | symbol'.`_

I don't see a `2360` in TypeScript's `diagnosticMessages.json`?